src/app/app-routing.module.ts

```diff
@@ -1,22 +1,23 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-import { HomeComponent } from './home/home.component';
-import { DiscoverComponent } from './discover/discover.component';
-import { ContatoComponent } from './contato/contato.component';
-import { LoginComponent } from './login/login.component';
-import { CadastroComponent } from './cadastro/cadastro.component';
+import { HomeComponent } from './pages/public/home/home.component';
+import { DiscoverComponent } from './pages/public/discover/discover.component';
+import { CadastroComponent } from './pages/private/cadastro/cadastro.component';
+import { LoginComponent } from './pages/private/login/login.component';
+import { ContatoComponent } from './pages/public/contato/contato.component';
+
 
 const routes: Routes = [
-    { path: '', component: HomeComponent },
     { path: 'discover', component: DiscoverComponent},
     { path: 'cadastro', component: CadastroComponent},
     { path: 'login', component: LoginComponent},
-    { path: 'contato', component: ContatoComponent}
+    { path: 'contato', component: ContatoComponent},
+    { path: '', component: HomeComponent }
```

Comment:
qdo vc usa o path:'' o ideal é usar tmb o parametro pathMatch: 'full'

```suggestion
    { path: '', pathMatch: 'full', component: HomeComponent }
```